apps/ai-content-generator/src/locations/Sidebar.jsx

```diff
@@ -0,0 +1,500 @@
+  }, [cma.contentType, cma.entry, richTextFields, sdk.entry, targetLocale]);
+
+  useMemo(() => {
+    const fields = sdk.entry.fields;
+    const textFields = Object.keys(fields)
+      .filter((key) => fields[key].type === 'Symbol' || fields[key].type === 'Text')
+      .filter((key) => fields[key].locales.includes(targetLocale));
+    setTextFields(
+      textFields.map((field) => {
+        return { name: field, currentEditor: true };
+      })
+    );
+
+    const richTextFields = Object.keys(fields)
+      .filter((key) => fields[key].type === 'RichText')
+      .filter((key) => fields[key].locales.includes(targetLocale));
+    setRichTextFields(
+      richTextFields.map((field) => {
+        return { name: field, currentEditor: true };
+      })
+    );
+
+    const translateableTextFields = Object.keys(fields).filter(
+      (key) =>
+        (fields[key].type === 'RichText' ||
+          fields[key].type === 'Symbol' ||
+          fields[key].type === 'Text') &&
+        fields[key]?.locales?.length > 1
+    );
+    setTranslateableTextFields(
+      translateableTextFields.map((field) => {
+        return { name: field, currentEditor: true, locales: fields[field].locales };
+      })
+    );
+  }, [sdk.entry.fields, targetLocale]);
+
+  function SetValueToField(field, value, locale = targetLocale) {
+    field.setValue(
+      field.type === 'RichText'
+        ? RichTextModel(value)
+        : value
+            .replace(/^\s+|\s+$/g, '')
+            .replace(/^['"]+|['"]+$/g, '')
+            .trim(), // trim whitespace and quotes
+      locale // always write this to the default locale
+    );
+  }
+
+  function getFieldName(id) {
+    return sdk.contentType.fields.find((field) => field.id === id)?.name || id;
+  }
+
+  async function Render(as = 'title', prompt = null) {
+    try {
+      let input = '';
+      setLoading(true);
+
+      if (as === 'translation') {
+        const sourceValue =
+          fields[field].type === 'RichText'
+            ? documentToPlainTextString(fields[field].getValue(sourceLocale))
+            : sdk.entry.fields[field].getValue(sourceLocale);
+        input = `Translate "${sourceValue}" to ${sdk.locales.names[targetLocale]}`;
+      } else {
+        input = prompts({ as });
+      }
+
+      const response = await Prompt(prompt ? input.replace('__prompt__', prompt) : input);
+      SetValueToField(sdk.entry.fields[field], response, targetLocale);
+      // Use the response coming back as an input prompt in the interface by default
+      setPrompt(response);
+      sdk.notifier.success('Done! Anything else?');
+    } catch (e) {
+      console.error(e);
+      sdk.notifier.error('Something went wrong. Please try again.');
+    } finally {
+      setLoading(false);
+      setOption(0);
+    }
+  }
+
+  return (
+    <>
+      <FormControl>
+        <FormControl.Label>Select an action</FormControl.Label>
+        <Select onChange={(event) => setOption(event.target.value)} value={option}>
```

Comment:
The outline on focus for each of these Select components is cut off on the sides (see screenshot)
![Screenshot 2023-04-03 at 11 05 09 AM](https://user-images.githubusercontent.com/62958907/229579292-903d9c58-1357-493b-8a92-02dbea91df04.png)

Maybe this is a small thing that could be adjusted?